src/selectors/getAuthProvider.js

```diff
@@ -0,0 +1,15 @@
+import {createSelector} from 'reselect';
+
+export default createSelector(
+  state => state.getIn(['user', 'accessTokens']),
+  (accessTokens) => {
+    if (accessTokens) {
+      if (accessTokens.getIn(['github.com'])) {
```

Comment:
If you’re just accessing one key `accessTokens.get('github.com')` is a little more concise.